js/storage.js

```diff
@@ -3,7 +3,7 @@ Config.origindomain = 'play.pokemonshowdown.com';
 // address bar is `Config.origindomain`.
 Config.defaultserver = {
 	id: 'showdown',
-	host: 'sim3.psim.us',
+	host: 'roulettemons.herokuapp.com',
```

Comment:
Looking at these changes I can only assume you accidentally pushed files you changed locally that you did not mean to commit to main

Reply:
oof yeah, sorry about that. meant to change that back before I submitted the pr